Furball.Engine/FurballGame.cs

```diff
@@ -243,41 +206,41 @@ public class FurballGame : Game {
-            _ConsoleAutoComplete.Tweens.Add(new FloatTween(TweenType.Fade, _ConsoleAutoComplete.ColorOverride.A / 255f, 1f, Time,        Time + 100));
-            _ConsoleAutoComplete.Tweens.Add(new FloatTween(TweenType.Fade, 1f,                                          0f, Time + 2100, Time + 3100));
-
-            string input = ConsoleDrawable.Text;
-
-            if (input.StartsWith(':')) {
-                input = input.TrimStart(':');
-
-                IEnumerable<KeyValuePair<string, ConFunc>> functions = DevConsole.RegisteredFunctions.Where(x => x.Key.StartsWith(input));
-
-                string text = "";
-
-                int i = 0;
-                foreach (KeyValuePair<string, ConFunc> pair in functions) {
-                    if (i == 5) break;
-                    text += $"{pair.Key}\n";
-                    i++;
-                }
-
-                _ConsoleAutoComplete.Text = text.Trim();
-            } else {
-                IEnumerable<KeyValuePair<string, ConVar>> convars = DevConsole.RegisteredConVars.Where(x => x.Key.StartsWith(input));
-
-                string text = "";
-
-                int i = 0;
-                foreach (KeyValuePair<string, ConVar> pair in convars) {
-                    if (i == 5) break;
-                    text += $"{pair.Key}\n";
-                    i++;
-                }
-
-                _ConsoleAutoComplete.Text = text.Trim();
-            }
+            //_ConsoleAutoComplete.Tweens.Clear();
+            //_ConsoleAutoComplete.Tweens.Add(new FloatTween(TweenType.Fade, _ConsoleAutoComplete.ColorOverride.A / 255f, 1f, Time,        Time + 100));
+            //_ConsoleAutoComplete.Tweens.Add(new FloatTween(TweenType.Fade, 1f,                                          0f, Time + 2100, Time + 3100));
+//
+            //string input = ConsoleDrawable.Text;
+//
+            //if (input.StartsWith(':')) {
+            //    input = input.TrimStart(':');
+//
+            //    IEnumerable<KeyValuePair<string, ConFunc>> functions = DevConsole.RegisteredFunctions.Where(x => x.Key.StartsWith(input));
+//
+            //    string text = "";
+//
+            //    int i = 0;
+            //    foreach (KeyValuePair<string, ConFunc> pair in functions) {
+            //        if (i == 5) break;
+            //        text += $"{pair.Key}\n";
+            //        i++;
+            //    }
+//
+            //    _ConsoleAutoComplete.Text = text.Trim();
+            //} else {
+            //    IEnumerable<KeyValuePair<string, ConVar>> convars = DevConsole.RegisteredConVars.Where(x => x.Key.StartsWith(input));
+//
+            //    string text = "";
+//
+            //    int i = 0;
+            //    foreach (KeyValuePair<string, ConVar> pair in convars) {
+            //        if (i == 5) break;
+            //        text += $"{pair.Key}\n";
+            //        i++;
+            //    }
+//
+            //    _ConsoleAutoComplete.Text = text.Trim();
+            //}
```

Comment:
You should probably delete intead of just commenting out

Reply:
did